### PR TITLE
Update globalsettings.md

### PR DESCRIPTION
**Pull request by Abdjulaziz**
Reffering to pr: https://github.com/umbraco/UmbracoDocs/pull/5139

### CODE REVIEW

**Comment by sofietoft** [comment]
Thanks for adding this @Abdjulaziz ! 💪 

I see some warnings on some long sentences in there - mind taking an extra look?
Happy to help, if you're unable to shorten the sentences.

**Comment by Abdjulaziz** [comment]
There we go 😄 
All checks passed 💪

File: 11/umbraco-cms/reference/configuration/globalsettings.md [hunk 1]
```diff
@@ -74,7 +74,9 @@ A comma-separated list of all the folders in your directory to be left alone by
 
 ### Timeout
 
-Configures the number of minutes without any requests being made before the Umbraco user will be required to re-login. Any backoffice request will reset the clock. The format is HH:MM:SS.
+Configure the session timeout to determine how much time can pass without a requests being made before the user is required to log in again. The session timeout format can be set as HH:MM:SS, any activity within the backoffice will reset the timer.
```
Comment (sofietoft) [inline 11/umbraco-cms/reference/configuration/globalsettings.md:77]:
```suggestion
Configure the session timeout to determine how much time without a request being made can pass before the user is required to log in again. The session timeout format needs to be set as `HH:MM:SS`. Any activity within the backoffice will reset the timer.
```

**Review by sofietoft — COMMENTED** [review]
I've made a tiny suggestion to make the first sentence a little easier to read!

Let me know what you think!

**Comment by Abdjulaziz** [comment]
Push it 😂

**Comment by sofietoft** [comment]
But, @Abdjulaziz , you didn't accept the changes 😭 You just resolved the comment.

**Comment by sofietoft** [comment]
Sure! I mean, **now** you are 😎

**Pull request merged**